src/test/java/fi/metatavu/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -65,6 +157,97 @@ private void printIds() throws IOException {
     for (ServiceChannelType type : ServiceChannelType.values()) {
       printServiceChannelList(type, serviceChannelFiles);
     }
+    
+    System.out.println("\n    public final static String[][] ORGANIZATION_SERVICES = {");
+    List<String> organizationServices = new ArrayList<>();
+
+    for (int i = 0; i < organizationFiles.length; i++) {
+      File organizationFile = organizationFiles[i];
+      organizationServices.add(i, printOrganizationServicesList(i, organizationFile));
+    }
+
+    System.out.println(StringUtils.join(organizationServices, ",\n"));
+    System.out.println("    };");
+    
+    List<EnumMap<ServiceChannelType, List<String>>> servicesTypeChannels = createServiceChannelsTypeMap(serviceFiles);
+
+    for (ServiceChannelType channelType : ServiceChannelType.values()) {
+      System.out.println(String.format("\n    public final static String[][] SERVICE_%s_CHANNELS = {", channelType));
+      
+      List<String> channelTypeIds = new ArrayList<>();
+      
+      for (int i = 0; i < servicesTypeChannels.size(); i++) {
+        StringBuilder typeIds = new StringBuilder();
+        
+        typeIds.append("      {");
+        
+        EnumMap<ServiceChannelType, List<String>> serviceTypeChannels = servicesTypeChannels.get(i);
+        List<String> idList = serviceTypeChannels.get(channelType);
+        if (idList != null) {
+          List<String> ids = serviceTypeChannels.get(channelType).stream()
+            .map(id -> String.format(" \"%s\"", id))
+            .collect(Collectors.toList());
+          typeIds.append(StringUtils.join(ids, ","));
+        }
+        
+        typeIds.append(" }");
+        
+        channelTypeIds.add(typeIds.toString());
+      }
+      
+      System.out.println(StringUtils.join(channelTypeIds, ",\n"));
+      System.out.println("    };");
+    }
+  }
+  
+  private EnumMap<ServiceChannelType, List<String>> createChannelTypeMap() {
+    EnumMap<ServiceChannelType, List<String>> result = new EnumMap<>(ServiceChannelType.class);
+    
+    Arrays.stream(ServiceChannelType.values()).forEach((type) -> {
+      result.put(type, new ArrayList<>());
+    });
+    
+    Arrays.stream(getServiceChannelsFolder().listFiles()).forEach((channelFile) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "channelFile" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)